### ISSUE TITLE
requesting specific shows errors out.

### ISSUE THREAD

**Issue by zyphermonkey** [tidusjar/Ombi#2914]
**Describe the bug**
I tried requesting Dragon Ball and Dragon Ball Z, but neither got sent to Sonarr. 

I tried requesting a new show to and it added the first season of Sherlock with no issues.

I wonder if this is an issue with the age of the show and it missing some field/attribute on imdb or else where.

**To Reproduce**
Request any amount of Dragon Ball or Dragon Ball Z.

**Expected behavior**
Request to be successful and be sent to Sonarr.

**Logs (Logs directory where Ombi is located)**
```
2019-04-06 10:25:21.349 -04:00 [Error] Exception thrown when sending a movie to DVR app, added to the request queue
System.ArgumentNullException: Value cannot be null.
Parameter name: s
   at Ombi.Core.Senders.TvSender.SendToSonarr(ChildRequests model, SonarrSettings s)
   at Ombi.Core.Senders.TvSender.Send(ChildRequests model) in C:\projects\requestplex\src\Ombi.Core\Senders\TvSender.cs:line 61

```

**Ombi Version (please complete the following information):**
 - Ombi Version 3.0.4256
- Sonarr Version 2.0.0.5322

**Comment by tidusjar** [comment]
Have you filled in your anime settings in sonarr?

**Comment by zyphermonkey** [comment]
I'm embarrassed, but I never knew there were different settings for anime. This is the first anime show I've ever added.

For those coming after me I added the following categories in sonarr for anime: 5070,5030,5040
I don't think the 5030 and 5040 are required, but I wanted them there as a fallback.

In Ombi I also had to set my quality profile and default root folder for anime under the sonarr settings.
